Makefile

```diff
@@ -7,7 +7,6 @@ DOCKER_COMPOSE := $(shell $(PROJECT_ROOT)/scripts/get-docker-compose.sh)
 .PHONY: all
 all:
 	$(DOCKER_COMPOSE) up --build -d
-	make seed
```

Comment:
After these changes, we can't run make seed here because the migrations take a while to execute.